base/applications/wusa/wusa.c

```diff
@@ -0,0 +1,19 @@
+/*
+* PROJECT:          ReactOS Setup
```

Comment:
Setup?

Reply:
yes, WUSA is a installer

Reply:
"ReactOS Setup" is already reserved name of an existing component (2nd stage setup).

Reply:
Ok

Reply:
ok